### DISCUSSION TITLE
External image in Next 12.3.4

### DISCUSSION THREAD

**Discussion by Rinat-Arifullin** [vercel/next.js#73129] [Help]
I have external image https://www.mos.ru/upload/newsfeed/afisha/3523b8e36276f97a695733c40d752bb2/MGCaTp5JPYy0.jpg
My next config 
`  
images: {
    domains: ['www.mos.ru', 'mosrucdn.mos.ru'],
    formats: ['image/webp', 'image/avif'],
  },
  `
I got 404 

same behavior np production
 
![image](https://github.com/user-attachments/assets/6922866d-4ee3-4c26-9362-4d867c289c74)

If i set unoptimized this is work, but i need optimize

**Comment by denis23x** [comment]
Hi @Rinat-Arifullin 

Try to add `image/jpeg` into `formats`, and `mos.ru` without `www` (just for check) into `domains`

Make sure your image is really accessible, for example your link doesn't load in my case (just hangs in loading)

**Reply by Rinat-Arifullin** [reply]
warn  - Invalid next.config.js options detected: 
  - The value at .images.formats[2] must be one of: "image/avif" or "image/webp"

**Reply by Rinat-Arifullin** [reply]
image work if i use` <img src={url} alt={alt}/>`

**Reply by denis23x** [reply]
Nah, I only getting 504 with `<Image>` and 502 with `<img>` with your url 🤷‍♂️ can't reproduce at all.

Try to check out your image storage, maybe there are some restrictions.

**Reply by Rinat-Arifullin** [reply]
I try other img, just from google https://upload.wikimedia.org/wikipedia/commons/thumb/7/77/Google_Images_2015_logo.svg/1200px-Google_Images_2015_logo.svg.png
![image](https://github.com/user-attachments/assets/12200cbe-c532-490c-b3ab-089dcd393473)
![image](https://github.com/user-attachments/assets/71024b79-e630-4ea5-b05d-6620847762f2)
![image](https://github.com/user-attachments/assets/4064d8ca-79ff-41a3-8391-ab3b52d6d92d)
the result is the same

**Reply by Rinat-Arifullin** [reply]
I use next 12.3.4 by the way
